index.src.html

```diff
@@ -345,15 +345,232 @@
+                </p>
+                <p>
+                  `Yd = dz * px`
+                </p>
+                <p>
+                  `Zd = dz`
+                </p>
+              </li>
+              <li>If <a>depth map</a>'s <a>deprojection distortion
+              coefficients</a> <a>k1</a>, <a>k2</a>, <a>k3</a>, <a>p1</a> and
+              <a>p2</a> are available, with a note that some of those could be
+              zero,
+                <p>
+                  3D coordinates (Xd, Yd, Zd) in depth camera space are
+                  calculated as:
+                </p>
+                <p>
+                  `r2 = px^2 + py^2`
+                </p>
+                <p>
+                  `r = 1 + k1 * r2 + k2 * r2^2 + k3 * r2^3`
+                </p>
+                <p>
+                  `Xd = dz * (px * r + 2 * p1 * px * py + p2 * (r2 + 2 *
+                  px^2))`
+                </p>
+                <p>
+                  `Yd = dz * (py * r + 2 * p2 * px * py + p1 * (r2 + 2 *
+                  py^2))`
+                </p>
+                <p>
+                  `Zd = dz`
+                </p>
+              </li>
+            </ul>
+          </li>
+          <li>Transform 3D points from depth camera 3D space to color camera 3D
+          space.
+            <p>
+              The 3D point (Xd, Yd, Zd) calculated in previous step is in depth
+              camera 3D coordinate system. To get the coordinates of the same
+              point in space, but in color camera 3D coordinate system, we use
+              matrix multiplication of <a>transformation from depth to
+              video</a> matrix by the (Xd, Yd, Zd) 3D point vector.
+            </p>
+            <p>
+              Let (Xc, Yc, Zc) be the output of this step - a 3D coordinates of
+              projected <a>depth map value</a> to color camera 3D space.
+            </p>
+            <p>
+              Let <var>M</var> be <a>transformation matrix</a> defined in
+              <a>depth map</a>'s <a>depthToVideoTransform</a> field.
+            </p>
+            <p>
+              To multiply 4x4 matrix by 3 element vector, we extend the 3D
+              vector by one element to 4 dimensional vector. After
+              multiplication, we use vector's x, y and z coordinates as the
+              result.
+            </p>
+            <p>
+              `((Xc), (Yc), (Zc)) = ([M] xx ((Xd), (Yd), (Zd), (1))).xyz`
+            </p>
+          </li>
+          <li>Finally, project from color camera 3D space to color frame 2D
+          pixels.
+            <p>
+              With this step, we conclude mapping of a <a>depth map</a> pixel
+              to a pixel value in color video frame.
+            </p>
+            <p>
+              For projection we use color track's <a>projection distortion
+              coefficients</a> <a>k1</a>, <a>k2</a>, <a>k3</a>, <a>p1</a> and
+              <a>p2</a> if available. The color track that corresponds to the
```

Comment:
It is defined under - rewrote this part so there is no confusion. "available" should probably be "defined".